### DISCUSSION TITLE
Do I need an internet connection?

### DISCUSSION THREAD

**Discussion by toplustar** [toplustar/SpookyShibaGameApp#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Xbird-sudo** [comment]
Hi, do I need an internet connection to use this plugin ?

**Reply by Xbird-sudo** [reply]
Nope, you just need the device to be reachable (on the same network for example).

**Reply by toplustar** [reply]
Nope, you just need the device to be reachable (on the same network for example).
